### PR TITLE
 Add dropdown button into navbar.html

### PR DESCRIPTION
**Pull request by macartur**

### CODE REVIEW

**Comment by macartur** [comment]
@diraol  I know about it, but I created a script in https://github.com/kytos/documentation/pull/48/files to get last tarballs based on each release and make a folder with all docs versions and activate the drop-down button.

The structure of docs folder will be:
   - bethania
       - kytos
       - python-openflow
   - chico
        - kytos
        - python-openflow

**Comment by diraol** [comment]
Great! I didn't know about this script. So it is everything ok (despite the Build errors rs).
Waiting the build to be fixed.

**Comment by beraldoleal** [comment]
@macartur tests... please.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -12,6 +12,7 @@ Added
 Changed
 =======
 - Raise ValueError if not using bytes (e.g. string) in BinaryData.
+- Changed docs to show a dropdown button with all python-openflow releases.
```
Comment (diraol) [inline CHANGELOG.rst]: Move this to the Added section with the text: `Versioned documentation.`

**Comment by diraol** [comment]
This PR was merged on another side Branch and will be merged here soon.

**Comment by beraldoleal** [comment]
@diraol If there is a side branch with a solution ok, but lets keep the issue open until the merge be effective on this master.

**Pull request closed without merging**